document/core/util/bikeshed_fixup.py

```diff
@@ -10,7 +10,38 @@
 
 def Main():
   data = open(sys.argv[1]).read()
+
+  # Don't add more than 3 levels to TOC.
   data = data.replace('<h5>', '<h5 class="no-toc">')
```

Comment:
You can use [`Max ToC Depth: 3`](https://tabatkins.github.io/bikeshed/#metadata-max-toc-depth) in your metadata to do this automatically.